src/planner/vectorfield/VectorFieldPlanner.cpp

```diff
@@ -141,6 +141,14 @@ std::unique_ptr<aikido::trajectory::Spline> planToEndEffectorOffset(
     std::chrono::duration<double> timelimit,
     planner::PlanningResult* planningResult)
 {
+  // ensure that no two planners run at the same time
+  if (metaskeleton->getNumBodyNodes() <= 0)
+  {
+      throw std::runtime_error("MetaSkeleton doesn't have any body nodes.");
```

Comment:
The code formatting check wouldn't be happy with 4-space indentation. :smile: Please run `make format` to format the code (make sure `clang-format-3.8` is installed).